modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -707,90 +1283,6 @@ private void _createFDSSortObjectDefinition(
-						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
-						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
-						_language.get(locale, "description"), "description",
-						false),
-					ObjectFieldUtil.createObjectField(
-						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
-						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
-						_language.get(locale, "rest-application"),
-						"restApplication", true),
-					ObjectFieldUtil.createObjectField(
-						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
-						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
-						_language.get(locale, "rest-endpoint"), "restEndpoint",
-						true),
-					ObjectFieldUtil.createObjectField(
-						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
-						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
-						_language.get(locale, "rest-schema"), "restSchema",
-						true),
-					ObjectFieldUtil.createObjectField(
-						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
-						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
-						_language.get(locale, "list-of-items-per-page"),
-						"listOfItemsPerPage", true),
-					ObjectFieldUtil.createObjectField(
-						ObjectFieldConstants.BUSINESS_TYPE_INTEGER,
-						ObjectFieldConstants.DB_TYPE_INTEGER, true, false, null,
-						_language.get(locale, "default-items-per-page"),
-						"defaultItemsPerPage", true),
-					ObjectFieldUtil.createObjectField(
-						ObjectFieldConstants.BUSINESS_TYPE_LONG_TEXT,
-						ObjectFieldConstants.DB_TYPE_CLOB, true, false, null,
-						_language.get(locale, "creation-actions-order"),
-						"fdsCreationActionsOrder", false),
-					ObjectFieldUtil.createObjectField(
-						ObjectFieldConstants.BUSINESS_TYPE_LONG_TEXT,
-						ObjectFieldConstants.DB_TYPE_CLOB, true, false, null,
-						_language.get(locale, "fields-order"), "fdsFieldsOrder",
-						false),
-					ObjectFieldUtil.createObjectField(
-						ObjectFieldConstants.BUSINESS_TYPE_LONG_TEXT,
-						ObjectFieldConstants.DB_TYPE_CLOB, true, false, null,
-						_language.get(locale, "filters-order"),
-						"fdsFiltersOrder", false),
-					ObjectFieldUtil.createObjectField(
-						ObjectFieldConstants.BUSINESS_TYPE_LONG_TEXT,
-						ObjectFieldConstants.DB_TYPE_CLOB, true, false, null,
-						_language.get(locale, "item-actions-order"),
-						"fdsItemActionsOrder", false),
-					ObjectFieldUtil.createObjectField(
-						ObjectFieldConstants.BUSINESS_TYPE_LONG_TEXT,
-						ObjectFieldConstants.DB_TYPE_CLOB, true, false, null,
-						_language.get(locale, "sorts-order"), "fdsSortsOrder",
-						false),
-					ObjectFieldUtil.createObjectField(
-						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
-						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
-						_language.get(locale, "default-visualization-mode"),
-						"defaultVisualizationMode", false)));
-
-		_objectDefinitionLocalService.publishSystemObjectDefinition(
-			userId, fdsViewObjectDefinition.getObjectDefinitionId());
-
-		return fdsViewObjectDefinition;
-	}
-
```

Comment:
When we kill this, it means that `LPD-15729` becomes the FF both for the "simplified" and the "renamed" data model. 

It's ok, just to keep in mind that during some time in master, `LPD-15729` was just the FF for the "simplified" data model. We may find people that experiences DSM/Fragment stop working because they created the "simplified" data model with the old object definition names.

For those folks, there is no choice but start over with new data model (DB clearing or manual deletion of object definitions). Fyi @herrardo in case you find someone asking

Reply:
Both are stories in the [LPD-15729 epic](https://liferay.atlassian.net/browse/LPD-15729), so it made sense to me to put them under one FF. The transitional case you are describing is true, but the impact risk should be minimal. Within this week, a customer would need to know to enable LPD-15729 and create new persistence, and we don't promote this anywhere. It seem to me very unlikely.